src/vrppdtw/sql/routing_vrppdtw.sql

```diff
@@ -25,17 +25,18 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 -----------------------------------------------------------------------
 --
 --
-create or replace function pgr_gsoc_vrppdtw(
-                sql text,
-                vehicle_num integer,
-                capacity integer,
-                OUT seq integer,
-                OUT rid integer,
-                OUT nid integer,
-                OUT cost integer
-                )
-returns setof record as
+CREATE OR REPLACE FUNCTION pgr_gsoc_vrppdtw(sql text, vehicle_num integer, capacity integer, max_itr integer,
```

Comment:
please, can you reformat so that each parameter is in one line
and, if you make instead of max_itr:

max_cycles INTEGER DEFAULT 30
 with the default the old behaviour will be kept